packages/graphql/lib/src/core/raw_operation_data.dart

```diff
@@ -31,7 +45,7 @@ class RawOperationData {
   String get operationName {
     // XXX there is a bug in the `graphql_parser` package, where this result might be
     // null event though the operation name is present in the document
-    _operationName ??= getOperationName(document);
+    _operationName ??= getLastOperationName(documentNode);
```

Comment:
Can we test if this bug is fixed?

Reply:
Removing the comment. It's not too helpful in describing the exact conditions of the bug.

If any bug in parser appears, we'll file a bug to `package:gql` and we'll fix it there.